web/crux/src/guards/uuid-params.validation.guard.ts

```diff
@@ -0,0 +1,38 @@
+import { BadRequestException, CanActivate, ExecutionContext, Injectable } from '@nestjs/common'
+import { Reflector } from '@nestjs/core'
+
+import validator from 'validator'
+
+/**
+ * UuidValidationGuard ensures that incoming requests contain valid UUIDs for
+ * specified parameters, and rejects the request if any of the UUIDs
+ * are invalid.
+ */
+@Injectable()
+export default class UuidValidationGuard implements CanActivate {
+  constructor(private reflector: Reflector) {}
+
+  async canActivate(context: ExecutionContext): Promise<boolean> {
+    const params = this.reflector.get<string[]>('uuidParams', context.getHandler())
```

Comment:
Please put 'uuidParams' to a const in the decorator file.

Reply:
For sure!